modules/ipi-install-network-requirements.adoc

```diff
@@ -10,6 +10,23 @@ Installer-provisioned installation of {product-title} involves several network r
 
 image::210_OpenShift_Baremetal_IPI_Deployment_updates_0122_2.png[Installer-provisioned networking]
 
+[id="network-requirements-ensuring-required-ports-are-open_{context}"]
+== Ensuring required ports are open
+
+Certain ports must be open for installer-provisioned installations to compelete successfully. In most cases, the installer will configure this for you. In certain situations, such as using separate subnets for far edge worker nodes, you must ensure that worker nodes can communicate with control plane nodes over these ports.
+
+.Required ports
+[options="header"]
+|====
+|Port|Description
+
+|`5050`, `5051`| The Ironic Inspector API runs on the control plane nodes and listens on port `5050` and on secured port `5051`. The Inspector API is responsible for hardware introspection, which collects information about the hardware characteristics of the bare metal hosts.
+
+|`6180`, `6183`| The baseboard management controllers (BMC) of the cluster nodes must be able to access the control plane nodes using port `6180` or secured port `6183`.
```

Comment:
let's say that 6183 is the preferred option and this is needed to get the inspection and deployment images